browser/src/control/Control.Sidebar.ts

```diff
@@ -29,6 +29,7 @@ class Sidebar extends SidebarBase {
 	onAdd(map: ReturnType<typeof L.map>) {
 		super.onAdd(map);
 		this.map.on('sidebar', this.onSidebar, this);
+		this.map.on('unocommandclick', this.onUNOCommandClick, this);
```

Comment:
this is never fired

Reply:
I’ve set a breakpoint, and it triggers. Any clue why your case isn’t firing the event?

Reply:
beacause local code looks different than uploaded to github? :)

I don't see `unocommandclick` fire   in the code. But I didn't test. I mean "I don't see it fired anywhere"

Reply:
Oops, I forgot to add a chunk patch. :)